pkg/webhook/admission/pod/metrics_aggregate_injector.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2021 The KServe Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pod
+
+import (
+	"github.com/kserve/kserve/pkg/constants"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	defaultPrometheusPort          = "8080"
+	PrometheusPortAnnotationKey    = "prometheus.io/port"
+	PrometheusPathAnnotationKey    = "prometheus.io/path"
+	PrometheusPathAnnotationValue  = "/metrics"
+	QueueProxyAggregateMetricsPort = "9088"
```

Comment:
I can move this to `constants` , if we think that would be a better place for them?